tests/test-theme.php

```diff
@@ -1,15 +1,25 @@
 <?php
 
-class ThemeTest extends \WP_UnitTestCase {
+class ThemeTest extends \WP_UnitTestCase
+{
+    use utilsTrait;
 
-	public function test_update_template_root() {
+    public function test_migrate_book_themes() {
+        $this->_book();
+        delete_option( 'pressbooks_theme_migration' );
+        \Pressbooks\Theme\migrate_book_themes();
+        $this->assertEquals( 1, get_option( 'pressbooks_theme_migration' ) );
+        \Pressbooks\Theme\migrate_book_themes();
+        $this->assertEquals( 2, get_option( 'pressbooks_theme_migration' ) );
+    }
```

Comment:
[Triggered on `init`](https://github.com/pressbooks/pressbooks/blob/2b217032b61be032b59ca44865fed6d5fd07f4be/hooks.php#L253) so two page loads required to migrate? 

See previous comment.